app/views/videocall/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= react_component("StartSession", props: @session_props, prerender: Rails.env.test?) %>
```

Comment:
If you have set up routing in app.jsx you don't need that. You are rendering it in main.html.erb not here.
Try removing it

Reply:
Removed